gallery_widgets/lib/features/button/widgets/material_raw_material_button.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:gallery_widgets/features/widgets/ticket_clip_path.dart';
+
+class MaterialRawMaterialButton extends StatelessWidget {
+  const MaterialRawMaterialButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        ClipPath(
+          clipper: TicketClipper(
+            borderRadius: 10,
+            yClipperStart: MediaQuery.sizeOf(context).width * 0.23,
+            yClipRadius: 8,
+          ),
+          child: MaterialButton(
+            onPressed: () {},
+            minWidth: MediaQuery.sizeOf(context).width,
+            color: Colors.grey,
+            child: const Padding(
+              padding: EdgeInsets.symmetric(
+                vertical: 7,
+              ),
+              child: Text('Material Button'),
+            ),
+          ),
+        ),
+        const SizedBox(height: 10),
+        // second
+        RawMaterialButton(
+          onPressed: () {},
+          constraints: BoxConstraints(
+            minWidth: MediaQuery.sizeOf(context).width,
+          ),
+          shape: RoundedRectangleBorder(
```

Comment:
we have the `borderRadius`  in the `TicketClipper`, so Do we need the `borderRadius` in the `RoundedRectangleBorder `?

Reply:
without RoundedRectangleBorder, the shadow of the button will not bordered

https://github.com/krish-do-goldenowl/training_minhnhat_dart_challenge/assets/139732686/72953100-21e1-4215-9d38-5bcceb05cb17